Client/multiplayer_sa/CMultiplayerSA_WidescreenFix.cpp

```diff
@@ -0,0 +1,730 @@
+        0x5842C6,            // Radar Trace 1
+        0x5876BC, 0x587733, 0x587916, 0x587A02, 0x587A92,
+    };
+
+    for (size_t i = 0; i < std::size(radarWidthAddresses); i++)
+    {
+        MemPut<float*>(radarWidthAddresses[i] + 0x2, &RadarWidth[i]);
+    }
+
+    for (size_t i = 0; i < std::size(radarHeightAddresses); i++)
+    {
+        MemPut<float*>(radarHeightAddresses[i] + 0x2, &RadarHeight[i]);
+    }
+
+    uintptr_t hudWidthAddresses[95] = {
+        0x58EB3F,            // 0 Clock
+        0x58EC0C,            // 1 Clock
+        0x58F55C,            // 2 Money
+        0x58F5F4,            // 3 Money
+        0x5892CA,            // 4 Info bars
+        0x58937E,            // 5 Info bars
+        0x58EE7E,            // 6 Info bars
+        0x58EEF4,            // 7 Info bars
+        0x589155,            // 8 Info bars
+        0x58EF50,            // 9 Info bars
+        0x58EFC5,            // 10 Info bars
+        0x58922D,            // 11 Info bars
+        0x58F116,            // 12 Info bars
+        0x58F194,            // 13 Info bars
+        0x58D92D,            // 14 Weapon icons
+        0x58D8C3,            // 15 Weapon icons
+        0x58F91C,            // 16 Weapon icons
+        0x58F92D,            // 17 Weapon icons
+        0x5894C5,            // 18 Ammo
+        0x5894E9,            // 19 Ammo
+        0x58F9D0,            // 20 Ammo
+        0x58FA5D,            // 21 Ammo
+        0x58F9F5,            // 22 Ammo
+        0x58FA8E,            // 23 Ammo
+        0x58DCB8,            // 24 Wanted
+        0x58DD00,            // 25 Wanted
+        0x58DD7E,            // 26 Wanted
+        0x58DF71,            // 27 Wanted
+        0x58DFE5,            // 28 Wanted
+        0x58B09F,            // 29 Vehicle names
+        0x58B13F,            // 30 Vehicle names
+        0x58AD3A,            // 31 Area names
+        0x58AD65,            // 32 Area names
+        0x58AE4A,            // 33 Area names
+        0x58C395,            // 34 Subs
+        0x58C41D,            // 35 Subs
+        0x58C4DC,            // 36 Subs
+        0x5896D8,            // 37 Stats box
+        0x589703,            // 38 Stats box
+        0x58990C,            // 39 Stats box
+        0x58986D,            // 40 Stats box
+        0x5897C3,            // 41 Stats box
+        0x589A16,            // 42 Stats box
+        0x589B2D,            // 43 Stats box
+        0x589C73,            // 44 Stats box
+        0x589D61,            // 45 Stats box
+        0x589E49,            // 46 Stats box
+        0x589F31,            // 47 Stats box
+        0x58A013,            // 48 Stats box
+        0x58A090,            // 49 Stats box
+        0x58A134,            // 50 Stats box
+        0x58C863,            // 51 SuccessFailed text
+        0x58D2DB,            // 52 MissionTitle text
+        0x58D459,            // 53 MissionTitle text
+        0x58CBC1,            // 54 WastedBusted text
```

Comment:
Stats box, mission title, wasted, busted, helptext are unused in MTA. Probably things like timers, menu system, OddJob, TripleSkip, CDarkel, CGarages are also unused in MTA. You should go through these addresses and leave only what makes sense for the MTA

Reply:
changed